bench/latency.cc

```diff
@@ -4,9 +4,19 @@
 
 #include <pthreadpool.h>
 
+#ifdef _WIN32
+#	include <sysinfoapi.h>
+#endif
 
 static void SetNumberOfThreads(benchmark::internal::Benchmark* benchmark) {
+#ifdef _WIN32
```

Comment:
Since this is a C++ source file, I suggest to instead use `std::thread::hardware_concurrency()` on all platforms